bucket_99/rofi/manifests/plist.single

```diff
@@ -1,39 +1,39 @@
-%%DATADIR-ON%%/themes/Adapta-Nokto.rasi
-%%DATADIR-ON%%/themes/Arc-Dark.rasi
-%%DATADIR-ON%%/themes/Arc.rasi
-%%DATADIR-ON%%/themes/DarkBlue.rasi
-%%DATADIR-ON%%/themes/Indego.rasi
-%%DATADIR-ON%%/themes/Monokai.rasi
-%%DATADIR-ON%%/themes/Paper.rasi
-%%DATADIR-ON%%/themes/Pop-Dark.rasi
-%%DATADIR-ON%%/themes/android_notification.rasi
-%%DATADIR-ON%%/themes/arthur.rasi
-%%DATADIR-ON%%/themes/blue.rasi
-%%DATADIR-ON%%/themes/c64.rasi
-%%DATADIR-ON%%/themes/dmenu.rasi
-%%DATADIR-ON%%/themes/glue_pro_blue.rasi
-%%DATADIR-ON%%/themes/gruvbox-common.rasi
-%%DATADIR-ON%%/themes/gruvbox-dark-hard.rasi
-%%DATADIR-ON%%/themes/gruvbox-dark-soft.rasi
-%%DATADIR-ON%%/themes/gruvbox-dark.rasi
-%%DATADIR-ON%%/themes/gruvbox-light-hard.rasi
-%%DATADIR-ON%%/themes/gruvbox-light-soft.rasi
-%%DATADIR-ON%%/themes/gruvbox-light.rasi
-%%DATADIR-ON%%/themes/lb.rasi
-%%DATADIR-ON%%/themes/paper-float.rasi
-%%DATADIR-ON%%/themes/purple.rasi
-%%DATADIR-ON%%/themes/sidebar.rasi
-%%DATADIR-ON%%/themes/solarized.rasi
-%%DATADIR-ON%%/themes/solarized_alternate.rasi
+%%DATADIR-ON%%share/rofi/themes/Adapta-Nokto.rasi
+%%DATADIR-ON%%share/rofi/themes/Arc-Dark.rasi
+%%DATADIR-ON%%share/rofi/themes/Arc.rasi
+%%DATADIR-ON%%share/rofi/themes/DarkBlue.rasi
+%%DATADIR-ON%%share/rofi/themes/Indego.rasi
+%%DATADIR-ON%%share/rofi/themes/Monokai.rasi
+%%DATADIR-ON%%share/rofi/themes/Paper.rasi
+%%DATADIR-ON%%share/rofi/themes/Pop-Dark.rasi
+%%DATADIR-ON%%share/rofi/themes/android_notification.rasi
+%%DATADIR-ON%%share/rofi/themes/arthur.rasi
+%%DATADIR-ON%%share/rofi/themes/blue.rasi
+%%DATADIR-ON%%share/rofi/themes/c64.rasi
+%%DATADIR-ON%%share/rofi/themes/dmenu.rasi
+%%DATADIR-ON%%share/rofi/themes/glue_pro_blue.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-common.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-dark-hard.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-dark-soft.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-dark.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-light-hard.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-light-soft.rasi
+%%DATADIR-ON%%share/rofi/themes/gruvbox-light.rasi
+%%DATADIR-ON%%share/rofi/themes/lb.rasi
+%%DATADIR-ON%%share/rofi/themes/paper-float.rasi
+%%DATADIR-ON%%share/rofi/themes/purple.rasi
+%%DATADIR-ON%%share/rofi/themes/sidebar.rasi
+%%DATADIR-ON%%share/rofi/themes/solarized.rasi
+%%DATADIR-ON%%share/rofi/themes/solarized_alternate.rasi
```

Comment:
You'll see this will fail.  I was saying to remove "%%DATADIR-ON%% everywhere.  You don't have any options set (nor should you, this was a terrible setup on freebsd)

Reply:
are you test-building with "ravenadm test rofi" ?